gc/base/standard/ConcurrentGCIncrementalUpdate.cpp

```diff
@@ -1,205 +1,1432 @@
+		cardTable->getCardTableStats()->getConcurrentCleanedCards(),
+		_stats.getCardCleaningThreshold(),
+		cardTable->getCardTableStats()->getCardCleaningPhase1Kickoff(),
+		cardTable->getCardTableStats()->getCardCleaningPhase2Kickoff(),
+		cardTable->getCardTableStats()->getCardCleaningPhase3Kickoff(),
+		_stats.getConcurrentWorkStackOverflowCount()
+	);
+}
+
+/**
+ * Create new instance of ConcurrentGCIncrementalUpdate object.
+ *
+ * @return Reference to new MM_ConcurrentGCIncrementalUpdate object or NULL
+ */
+MM_ConcurrentGCIncrementalUpdate *
+MM_ConcurrentGCIncrementalUpdate::newInstance(MM_EnvironmentBase *env)
+{
+	MM_ConcurrentGCIncrementalUpdate *concurrentGC =
+			(MM_ConcurrentGCIncrementalUpdate *)env->getForge()->allocate(sizeof(MM_ConcurrentGCIncrementalUpdate),
+					OMR::GC::AllocationCategory::FIXED, OMR_GET_CALLSITE());
+
+	if (NULL != concurrentGC) {
+		new(concurrentGC) MM_ConcurrentGCIncrementalUpdate(env);
+		if (!concurrentGC->initialize(env)) {
+			concurrentGC->kill(env);
+			concurrentGC = NULL;
+		}
+	}
+
+	return concurrentGC;
+}
+
+/**
+ * Destroy instance of an ConcurrentGCIncrementalUpdate object.
+ *
+ */
+void
+MM_ConcurrentGCIncrementalUpdate::kill(MM_EnvironmentBase *env)
+{
+	tearDown(env);
+	env->getForge()->free(this);
+}
+
+
+/* Teardown a ConcurrentGCIncrementalUpdate object
+* Destroy referenced objects and release
+* all allocated storage before ConcurrentGCIncrementalUpdate object is freed.
+*/
+void
+MM_ConcurrentGCIncrementalUpdate::tearDown(MM_EnvironmentBase *env)
+{
+	if (NULL != _cardTable) {
+		_cardTable->kill(env);
+		_cardTable= NULL;
+		_extensions->cardTable = NULL;
+	}
+
+	/* ..and then tearDown our super class */
+	MM_ConcurrentGC::tearDown(env);
+}
+
+/**
+ * Initialize a new ConcurrentGCIncrementalUpdate object.
+ * Instantiate card table and concurrent RAS objects(if required) and initialize all
+ * monitors required by concurrent. Allocate and initialize the concurrent helper thread
+ * table.
+ *
+ * @return TRUE if initialization completed OK;FALSE otherwise
+ */
+bool
+MM_ConcurrentGCIncrementalUpdate::initialize(MM_EnvironmentBase *env)
+{
+	J9HookInterface** mmPrivateHooks = J9_HOOK_INTERFACE(_extensions->privateHookInterface);
+
+	if (!MM_ConcurrentGC::initialize(env)) {
+		goto error_no_memory;
+	}
+
+	if (!createCardTable(env)) {
+		goto error_no_memory;
+	}
+
+	/* Register on any hook we are interested in */
+	(*mmPrivateHooks)->J9HookRegisterWithCallSite(mmPrivateHooks, J9HOOK_MM_PRIVATE_CARD_CLEANING_PASS_2_START, hookCardCleanPass2Start, OMR_GET_CALLSITE(), (void *)this);
+
+	_secondCardCleanPass = (_extensions->cardCleaningPasses == 2) ? true : false;
+	_allocToTraceRateCardCleanPass2Boost = _extensions->cardCleanPass2Boost;
+
+	_cardCleaningFactorPass1 = MM_ConcurrentGC::interpolateInRange(INITIAL_CARD_CLEANING_FACTOR_PASS1_1,
+												INITIAL_CARD_CLEANING_FACTOR_PASS1_8,
+												INITIAL_CARD_CLEANING_FACTOR_PASS1_10,
+												_allocToTraceRateNormal);
```

Comment:
personally I am not in favour of this kind of attempts to align parameters - it usually does not work if source code environment is slightly different. IMO formatting in two lines or even one long line is better

Reply:
Fixed - formatted into single line

Reply:
I think I got them all in these files